extensions/windowgram/init.lua

```diff
@@ -0,0 +1,210 @@
+--- === hs.windowgram ===
+---
+--- region generation using ascii rows similar to tmuxomatic for window areas
```

Comment:
This line is what will be used as the module summary at http://www.hammerspoon.org/docs/ - it should try to sell itself in a single sentence. "region generation" is a bit too vague IMO. I think something more like the original description would be better here - the majority use case for end users, would be to use this for tuxomatic style window management.